internal/api/connectors/cloudConnector.go

```diff
@@ -81,6 +84,18 @@ func NewConnectorClient(cfg *viper.Viper) CloudConnectorClient {
 	return NewConnectorClientWithHttpRequestDoer(cfg, &httpClient)
 }
 
+func encodedBody(body PostMessageJSONRequestBody) (io.Reader, error) {
+	var bodyReader io.Reader
+	buf := &bytes.Buffer{}
+	encoder := json.NewEncoder(buf)
+	encoder.SetEscapeHTML(false)
+	if err := encoder.Encode(body); err != nil {
+		return nil, err
+	}
+	bodyReader = bytes.NewReader(buf.Bytes())
+	return bodyReader, nil
```

Comment:
`&bytes.Buffer{}` does itself implement `io.Reader` so you can just do `return buf, nil`

Reply:
Done! :+1: